### PR TITLE
Clear information about overwriting configuration keys

### PR DESCRIPTION
**Pull request by benjaoming**
# Your checklist for this pull request

#1480

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or in [Jenkins JIRA](https://issues.jenkins-ci.org)
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### CODE REVIEW

**Review by timja — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -124,6 +124,8 @@ If `CASC_JENKINS_CONFIG` points to a folder, the plugin will recursively travers
 :x: `/jenkins/.configs/casc_configs/..dir2/config.yaml`  
 </details>
 
+Configuration file sets traversed in folders cannot overwrite configuration values, this will create a conflict and raise a `ConfiguratorException`. Thus, the order of traversal does not matter to the final outcome.
```
Comment (benjaoming) [inline README.md:127]:
```suggestion
All configuration files that are discovered MUST be supplementary. They cannot overwrite each other's configuration values. This creates a conflict and raises a `ConfiguratorException`. Thus, the order of traversal does not matter to the final outcome.
```

Comment (benjaoming) [reply README.md:127]: Feel free to use this alternative phrasing if it seems more clear.

Comment (oleg-nenashev) [reply README.md:127]: +1 for alternate version. Thanks!

**Review by oleg-nenashev — APPROVED** [review]

**Comment by timja** [comment]
oops forgot to merge this

**Pull request merged**